lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -339,6 +368,21 @@ definitions:
       active_apns:
         $ref: '#/definitions/apn_list'
 
+  paginated_subscribers:
+    description: Page of subscribers
+    type: object
+    required:
+      - next_page_token
+      - subscribers
+    properties:
+      next_page_token:
+        $ref: '#/definitions/next_page_token'
+      subscribers:
+        type: object
+        additionalProperties:
+          x-nullable: true
```

Comment:
Why `x-nullable` for the map's values?

Reply:
Existing subscriber endpoint has nullable subscribers in its map, so all functions return a map of pointers to subs. We can update v2 for this but will require a bit of uncleanliness to do the conversion.